### PR TITLE
chore: have CI for tests across all Os's (manual) while default CI ru…

### PR DESCRIPTION
**Pull request by danielpeintner**
…ns on linux only

alternative for https://github.com/eclipse/thingweb.node-wot/pull/865

### CODE REVIEW

**Comment by danielpeintner** [comment]
@JKRhb Sorry for pinging you again.

```
   1) CoAP server implementation
       should support TD Content-Format negotiation:
     Error: Unknown Content-Format: application/foobar
```

Have you seen such issues on your side?
https://github.com/eclipse/thingweb.node-wot/actions/runs/3530516899/jobs/5922593923#step:7:195

I am seeing it in *this* PR and also in https://github.com/eclipse/thingweb.node-wot/pull/865
https://github.com/eclipse/thingweb.node-wot/actions/runs/3522562908/jobs/5928441681#step:7:195

**Comment by JKRhb** [comment]
Oh, sorry, using that fictitious content type must have broken something :/ I'll look into it!

**Comment by JKRhb** [comment]
Should be fixed in #879 now :)

File: .github/workflows/ci-full.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
+name: Full CI Pipeline
+
+on: workflow_dispatch
```
Comment (JKRhb) [inline .github/workflows/ci-full.yaml:3]: Maybe we could also let the full CI pipeline only run for changes on the main branch instead? This way it does not slow down progress in PRs but still uncovers OS-specific errors.

Comment (danielpeintner) [reply .github/workflows/ci-full.yaml:3]: At the moment the *full* test is not run automatically. Hence one needs to trigger it manually. When doing so once can usually select the branch it should run this action.. this is at least how I have been seeing this working.

Do you mean something else ?
Do you mean we should use `on: [push]` only and remove `pull_request` ?

Comment (JKRhb) [reply .github/workflows/ci-full.yaml:3]: > Do you mean something else ?
> Do you mean we should use on: [push] only and remove pull_request ?

Exactly, that is what I meant :)

Comment (danielpeintner) [reply .github/workflows/ci-full.yaml:3]: Makes sense!
@relu91 any thoughts?

**Comment by danielpeintner** [comment]
Note: firestore tests fail on ubuntu_latest also.
Hence, I will remove firestore from the test set.

**Review by relu91 — APPROVED** [review]
LGTM... just remember to have a tracking issue so that in the future we can restore the full checklist.

**Pull request merged**